source/frontend/src/components/layout/Header/UserProfile.tsx

```diff
@@ -19,14 +19,14 @@ const StyleDropDown = styled(NavDropdown)`
     padding: 0.1rem;
   }
   .dropdown-item {
-    background-color: ${variables.primaryLightColor};
+    background-color: ${props => props.theme.bcTokens.surfaceColorBackgroundDarkBlue};
     border-top: 0.2rem solid ${variables.accentColor};
```

Comment:
**nit:** `${variables.accentColor};` should be changed to `${props} => props.theme.css.whateverColor` and the import for **_variables.scss** removed